patches/0005-LPS-112982-Add-additional-resource-URL-parameters.patch

```diff
@@ -0,0 +1,71 @@
+From 1551d5d3ab5926fcf22e78837da2fac279b6177b Mon Sep 17 00:00:00 2001
+From: Julien Castelain <julien.castelain@liferay.com>
+Date: Mon, 6 Jul 2020 11:55:54 +0200
+Subject: [PATCH 5/5] LPS-112982 Add additional resource URL parameters
+
+---
+ core/ckeditor_base.js | 25 ++++++++++++++++++-------
+ core/config.js        |  9 +++++++++
+ 2 files changed, 27 insertions(+), 7 deletions(-)
+
+diff --git a/core/ckeditor_base.js b/core/ckeditor_base.js
+index 3fb18f5f7..e283f2011 100644
+--- a/core/ckeditor_base.js
++++ b/core/ckeditor_base.js
+@@ -171,15 +171,26 @@ if ( !window.CKEDITOR ) {
+ 			 * @returns {String} The full URL.
+ 			 */
+ 			getUrl: function( resource ) {
+-				// If this is not a full or absolute path.
+-				if ( resource.indexOf( ':/' ) == -1 && resource.indexOf( '/' ) !== 0 )
+-					resource = this.basePath + resource;
++				var resourceUrl = resource.includes(CKEDITOR.basePath)
++						? new URL(resource)
++						: new URL(CKEDITOR.basePath + resource);
+ 
+-				// Add the timestamp, except for directories.
+-				if ( this.timestamp && resource.charAt( resource.length - 1 ) != '/' && !( /[&?]t=/ ).test( resource ) )
+-					resource += ( resource.indexOf( '?' ) >= 0 ? '&' : '?' ) + 't=' + this.timestamp;
++				if (!resourceUrl.toString().endsWith('/')) {
++					if (!resourceUrl.searchParams.has('t')) {
++						resourceUrl.searchParams.set('t', CKEDITOR.timestamp);
```

Comment:
Just to repeat that comment from the earlier version of this commit which might have gotten lost: old code only did this after checking that `this.timestamp` was non-falsey. I would imagine that we should maintain that check, or you could explain the reason for not maintaining it.

Reply:
Yes I didn't understand all of it, we do need to check that `this.timestamp` is not falsy